python/tk_multi_publish2/publish_tree_widget/custom_widget_base.py

```diff
@@ -111,6 +112,13 @@ def set_status(self, status):
         if status == self.NEUTRAL:
             self.ui.status.hide()
         else:
+            if len(message) > 0 :               
+               if info_below:
+                  message += "\n See below or click for details."
```

Comment:
I think that extra space after the \n screws up the text.

<img width="51" alt="screen shot 2017-10-02 at 11 47 23" src="https://user-images.githubusercontent.com/8126447/31086302-7d4fca84-a767-11e7-9d53-339b20976ad5.png">